<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODBC_QueryClass</class>
 <widget class="QMainWindow" name="ODBC_QueryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODBC_Query</string>
  </property>
  <property name="windowIcon">
   <iconset resource="odbc_query.qrc">
    <normaloff>:/ODBC_Query/Resources/database_icon.ico</normaloff>:/ODBC_Query/Resources/database_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="StatusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="MainHorizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="HorizontalSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="TableTreeWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="FieldTreeWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>75</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QSplitter" name="VerticalSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="ODBC_TableView" name="SQLResultTableView">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <layout class="QHBoxLayout" name="HorizontalLayout_3" stretch="0,0,0,0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="ExecuteToolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;F5&gt; Executes the entered SQL statement</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="odbc_query.qrc">
             <normaloff>:/ODBC_Query/Resources/play_icon.png</normaloff>:/ODBC_Query/Resources/play_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="Line">
           <property name="windowModality">
            <enum>Qt::NonModal</enum>
           </property>
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="LeftToolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>left</string>
           </property>
           <property name="icon">
            <iconset resource="odbc_query.qrc">
             <normaloff>:/ODBC_Query/Resources/left_arrow.png</normaloff>:/ODBC_Query/Resources/left_arrow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CurrentStatementLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="RightToolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>right</string>
           </property>
           <property name="icon">
            <iconset resource="odbc_query.qrc">
             <normaloff>:/ODBC_Query/Resources/right_arrow.png</normaloff>:/ODBC_Query/Resources/right_arrow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="Line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="HorizontalLayout">
         <item>
          <widget class="QTextEdit" name="SQLCommandTextEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="SQLLogTextBrowser">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="MenuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <widget class="QMenu" name="MenuOpen_connections">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Open connections...</string>
     </property>
    </widget>
    <addaction name="ActionNew_Connection"/>
    <addaction name="MenuOpen_connections"/>
    <addaction name="ActionClose_all_connections"/>
   </widget>
   <addaction name="MenuFile"/>
   <addaction name="MenuConnections"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="ActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="ActionNew_Connection">
   <property name="text">
    <string>New connection</string>
   </property>
  </action>
  <action name="ActionClose_all_connections">
   <property name="text">
    <string>Close all connections</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ODBC_TableView</class>
   <extends>QTableView</extends>
   <header>ODBC_TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="odbc_query.qrc"/>
 </resources>
 <connections/>
</ui>
